sdk/core/AzureCore/Tests/CollectionsTests.swift

```diff
@@ -60,7 +60,7 @@ class CollectionsTests: XCTestCase {
             baseUrl: URL(string: "http://www.microsoft.com")!,
             transport: URLSessionTransport(),
             policies: [
-                UserAgentPolicy(sdkName: "Test", sdkVersion: "1.0")
+                UserAgentPolicy(sdkName: "Test", sdkVersion: "1.0", telemetryOptions: TelemetryOptions())
```

Comment:
Making `telemetryOptions` optional would eliminate the need to do this here and throughout.

Reply:
It's true, but I felt it was important enough to make it a required parameter. Again, it's basically just a judgement call.